multicluster/build/Dockerfile

```diff
@@ -0,0 +1,27 @@
+# Build the manager binary
```

Comment:
perhaps include them under build/images/* as is done for parent repo. unless all your yamls will be under parent repo build/yamls/

Reply:
this Kube-builder specific.